packages/zowe-explorer/webviews/edit-attributes/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "edit-attributes",
+  "private": true,
+  "type": "module",
+  "version": "1.0.0",
```

Comment:
I know this is not intended por publishing as a standalone app, but should the version match what we have in our `<root>/lerna.json`? or is it ok to remove it entirely?

Reply:
I've updated the version to match Zowe Explorer - but, this won't be updated automatically as its not directly in the `packages` folder, so we'll have to update it when the version changes 😬